src/NSwag.CodeGeneration/Models/ResponseModelBase.cs

```diff
@@ -71,6 +71,9 @@ public bool IsDate
             }
         }
 
+        /// <summary>Gets a value indicating whether this is a text/plain response.</summary>
+        public bool IsPlain => _response.Content.ContainsKey("text/plain") || _operationModel.Produces == "text/plain";
```

Comment:
Maybe rename to IsPlainText?

Reply:
Does this add support for Swagger 2.0 and OAI 3.0?

Reply:
the tests pass for both and I'm using 2 ways of specifying the response content type: inline (in 200 response) and top-level "produces". Does it cover all the angles? - I'm not sure, but it's a start.